### PR TITLE
fix: set merge conflict Action to run every 12 hours instead of every 15 minutes

### PR DESCRIPTION
**Pull request by RandellDawson**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x]  My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Related to PR #39327

After a discussion with @ojeytonwilliams, we decided that running this Action every 12 hours is sufficient for labeling/unlabeling  merge conflicts.

### CODE REVIEW

File: .github/workflows/label_merge_conflicts.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: 'Check for merge conflicts'
 on:
   schedule:
     # run this Action every 15 minutes
```
Comment (ojeytonwilliams) [inline .github/workflows/label_merge_conflicts.yml:4]:
```suggestion
    # run this Action every 12 hours
```

File: .github/workflows/label_merge_conflicts.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: 'Check for merge conflicts'
...
-    - cron:  '*/15 * * * *'
+    - cron:  '* */12 * * *'
```
Comment (ojeytonwilliams) [inline .github/workflows/label_merge_conflicts.yml:5]:
```suggestion
    - cron:  '0 */12 * * *'
```
If there's a * in the first column, it'll fire every minute in the 0th and 12th hours.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@ojeytonwilliams Sorry about that crontab mistake.

**Comment by ojeytonwilliams** [comment]
Nothing to worry about.  Even if I'd missed it, it wouldn't have been a big problem.

**Review by ojeytonwilliams — APPROVED** [review]

**Pull request merged**